material/plugins/social/plugin.py

```diff
@@ -76,10 +87,20 @@ def on_config(self, config):
             return
 
         # Check dependencies
-        if "Image" not in globals():
+        if DEP_IMPORT_ERRORS:
+            raise PluginError(
+                "Required dependencies of \"social\" plugin not found.\n"
+                "Install with: pip install \"mkdocs-material[imaging]\"\n"
+                + str("\n".join(DEP_IMPORT_ERRORS))
+            )
+
+        if CAIROSVG_ERROR:
             raise PluginError(
-                "Required dependencies of \"social\" plugin not found. "
-                "Install with: pip install \"mkdocs-material[imaging]\""
+                "\"cairosvg\" Python module is installed, but it crashed with:\n"
```

Comment:
What if we create a troubleshooting section in our image processing documentation and link to it? I think this might be a better approach because we have essentially infinite space as opposed to a terminal, it is easier to maintain and format, and we can cater to more OSs.

We then would only include the link here.

Reply:
OK, I will write up a small troubleshooting guide and put a link. Should I use the t.ly link shortener again?

Reply:
Yes, [t.ly](https://t.ly/) is the one we're using because it seems to be the most sensible to use right now.